types/google-apps-script/google-apps-script.spreadsheet.d.ts

```diff
@@ -2375,6 +2428,24 @@ declare namespace GoogleAppsScript {
             ): SpreadsheetTheme;
             setFontFamily(fontFamily: string): SpreadsheetTheme;
         }
+        /**
+         * An enumerations representing the sort order.
+         */
+        enum SortOrder {
+            ASCENDING,
+            DESCENDING,
+        }
+        /**
+         * The sorting specification.
+         */
+        interface SortSpec{
+            getBackgroundColor(): Color;
+            getDataSourceColumn(): DataSourceColumn;
+            getDimensionIndex(): Integer;
+            getForegroundColor(): Color;
+            getSortOrder(): SortOrder;
+            isAscending(): boolean;
```

Comment:
```suggestion
            getBackgroundColor(): Color | null;
            getDataSourceColumn(): DataSourceColumn;
            getDimensionIndex(): number | null;
            getForegroundColor(): Color | null;
            getSortOrder(): SortOrder;
            isAscending(): boolean;
```
https://developers.google.com/apps-script/reference/spreadsheet/sort-spec